src/ethereum_spec_tools/evm_tools/loaders/fixture_loader.py

```diff
@@ -0,0 +1,180 @@
+"""
+Defines Load class for loading json fixtures for the evm
+tools (t8n, b11r, etc.) as well as the execution specs
+testing framework.
+"""
+
+from abc import ABC, abstractmethod
+from typing import Any, Tuple
+
+from ethereum import rlp
+from ethereum.base_types import U256
+from ethereum.crypto.hash import Hash32
+from ethereum.utils.hexadecimal import (
+    hex_to_bytes,
+    hex_to_bytes8,
+    hex_to_bytes32,
+    hex_to_hash,
+    hex_to_u64,
+    hex_to_u256,
+    hex_to_uint,
+)
+
+from .fork_loader import ForkLoad
+from .transaction_loader import TransactionLoad
+
+
+class BaseLoad(ABC):
+    """Base class for loading json fixtures"""
```

Comment:
```suggestion
    "Base class for loading JSON fixtures"
```

Can use one double quote for single line docstrings.